### ISSUE TITLE
Does not build with latest serd 1.x

### ISSUE THREAD

**Issue by loki42** [drobilla/python-serd#2]
Unsure which version of python-serd builds with which commit of serd 1.x, current python-serd depends on removed statement code.

**Comment by drobilla** [comment]
Right, sorry, I should have kept a working tag or branch around.  Still trying to lock down the API but have been too short on focus time to keep all the parts of such an ambitious thing on the rails at once.  I'm traveling right now but I'll try to get them back in sync next week

**Comment by loki42** [comment]
All good,  there's a branch that's not in the github but is in gitlab that
it does build against.  I'm trying to fix a few bugs in Ingen so want to be
running the latest everything so fixes are easy to share.

**Comment by sven-h** [comment]
Hi @drobilla, Hi @loki42 ,

I also want to use python-serd but was unable to compile python-serd with any commit of the 1.x branch.
Where exactly is a version that works with the latest commit of python-serd?
Thank you very much.

Best regards
Sven

**Comment by drobilla** [comment]
I don't even know and have some work to do before the two will be in sync again.
